source/blog/2016-05-19-introducing-subteams.md

```diff
@@ -5,7 +5,9 @@ tags: Recent Posts
 ---
 
 I'm very happy to share with you our plan to scale Ember's day-to-day
-decision-making and coordination–something we're calling Subteams.
+decision-making and coordination–something we're calling Subteams. We mentioned
+some of this in this year's EmberConf keynote, but wanted to expand on the
+specific details.
```

Comment:
`and` seems better than `but`, you're not contradicting yourself :) Looks otherwise great!